extension/js/common/inject.ts

```diff
@@ -46,8 +46,7 @@ export class Injector {
     this.S = Ui.buildJquerySels({ // these are selectors that are not specific to any webmail variant
       body: 'body',
       compose_button_container: this.container.composeBtnSel[this.webmailName],
-      compose_button: 'div.new_message_button',
-      compose_button_label: '#cryptup_compose_button_label',
```

Comment:
I removed `compose_button_label` because I didn't found its usage in the codebase